Sloth.Web/Startup.cs

```diff
@@ -125,6 +125,12 @@ public void ConfigureServices(IServiceCollection services)
                     o.SerializerSettings.ReferenceLoopHandling = ReferenceLoopHandling.Ignore;
                     o.SerializerSettings.Converters.Add(new StringEnumConverter());
                 });
+
+#if DEBUG
+            // Enable recompiling views without restarting app
+            services.AddControllersWithViews()
+                .AddRazorRuntimeCompilation();
```

Comment:
oh this is a good one -- i thought that changed recently and the app was restarting a lot more on peaks when i was changing things.